java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/dataaccess/impl/dao/BookingDaoImpl.java

```diff
@@ -43,7 +44,7 @@ public BookingDaoImpl() {
 
     BookingEntity booking = Alias.alias(BookingEntity.class);
     EntityPathBase<BookingEntity> alias = Alias.$(booking);
-    JPAQuery query = new JPAQuery(getEntityManager()).from(alias);
+    JPAQuery query = (JPAQuery) new JPAQuery(getEntityManager()).from(alias);
```

Comment:
This cast looks pointless to me. Also JPAQuery is now generified. Should IMHO be `JPAQuery<BookingEntity>`